src/frontend/src/components/public/PIAFormTabs/review/index.tsx

```diff
@@ -72,6 +72,19 @@ const PIAReview = ({ printPreview }: IReviewProps) => {
   const [reviewNote, setReviewNote] = useState<string>(
     pia?.review?.mpo?.reviewNote || '',
   );
+
+  const disableConfirmButton = () => {
+    if (pia.hasAddedPiToDataElements === false && reviewNote.trim() === '')
+      return true;
+    return false;
+  };
+  const enableMPOReviewViewMode = () => {
+    // if the user already done review the MPO review field, we will show
+    // the review result page, otherwise the app will show review page
+    // display a checkbox and ask the MPO to accept the accountability and add review note
+    if (pia?.review?.mpo?.reviewedAt && editReviewNote === false) return true;
```

Comment:
For PI isn't the review note optional? what if it is false but the acknowledge is checked. Should we be in view mode or read mode? Will run your code locally to test but this logic is NOT very clear.

Reply:
yes, for PI PIA, it allows users just check the checkbox without inputting any review note, and if they click confirm button, will send to the backend and there will be a reviewedAt timestamp that attaches to the object. but if the user never checks the checkbox, the reivewedAt will not exist.  so if the reviewedAt field have a value, we show the review result otherwise show review

Reply:
Please could you add this logic as part of the comment. It makes it a lot more clearer now.